### PR TITLE
RI-5999 new splashscreen for Redis Copilot

### PR DESCRIPTION
**Pull request by kchepikava**

### CODE REVIEW

File: redisinsight/ui/src/components/side-panels/components/copilot-panel/components/copilot-preview/CopilotPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+interface AiMessageWrapperProps {
+  children: React.ReactElement
+}
+
+const AiAnswerMessageWrapper = ({ children }: AiMessageWrapperProps) => (
+  <div className={styles.answerWrapper}>
+    <div className={styles.avatarWrapper}>
+      <div className={styles.chatbotAvatar}>
+        <EuiIcon aria-label="ai message icon" type={LogoSVG} color="#ffffff" />
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/side-panels/components/copilot-panel/components/copilot-preview/CopilotPreview.tsx:16]: is this color will be the same for the light theme?

File: redisinsight/ui/src/components/side-panels/components/copilot-panel/components/copilot-preview/CopilotPreview.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  </div>
+)
+
+const CopilotPreview = () =>
+  (
+    <div className={styles.wrapper}>
+      <div
+        className={styles.history}
+      >
+        <AiQuestionMessageWrapper>
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/side-panels/components/copilot-panel/components/copilot-preview/CopilotPreview.tsx:46]: I would suggest to create an array of React nodes and iterate throw it

File: redisinsight/ui/src/components/side-panels/components/copilot-panel/components/copilot-preview/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+.chatbotAvatar {
+  width: 30px;
+  height: 30px;
+  min-width: 30px;
+  border-radius: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #FF4438;
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/side-panels/components/copilot-panel/components/copilot-preview/styles.module.scss:75]: Do we need to put this color as variable?

File: redisinsight/ui/src/components/side-panels/components/copilot-panel/components/copilot-splash-screen/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    height: 100%;
+    width: 100%;
+
+    display: flex;
+    flex-direction: column;
+
+    .header {
+      flex-shrink: 0;
+      padding-left: 20px;
+      margin-top:20px;
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/side-panels/components/copilot-panel/components/copilot-splash-screen/styles.module.scss:20]: space

File: redisinsight/ui/src/constants/storage.ts [hunk 5]
```diff
@@ -35,7 +35,8 @@ enum BrowserStorageItem {
   capability = 'capability',
   aiChatSession = 'aiChatSession',
   selectedAiChat = 'selectedAiChat',
-  generalChatAgreements = 'generalChatAgreements'
+  generalChatAgreements = 'generalChatAgreements',
+  hideCopilotSplashScreen = 'hideCopilotSplashScreen',
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/constants/storage.ts:39]: I would suggest to combine these storage items to one (copilot) with object value

**Comment by kchepikava** [comment]
part of RI-6101

**Pull request closed without merging**